src/main/java/com/sumindaa/demo/employeeservice/model/Employee.java

```diff
@@ -3,10 +3,10 @@
 import java.util.Date;
 
 public class Employee {
-    private int id;
+  //  private int id;
```

Comment:
why id removed

Reply:
to get simplified :)
 . thought later can add the ID